src/main/groovy/com/ullink/gradle/sonar/SonarScannerMsbuildPlugin.groovy

```diff
@@ -18,12 +21,19 @@ class SonarScannerMsbuildPlugin implements Plugin<Project> {
     static final SONAR_SCANNER_EXE = 'SonarScanner.MSBuild.exe'
     // keep the 'sonarqube' name for compatibility with the org.sonarqube plugin
     static final SONAR_SCANNER_TASK_NAME = 'sonarqube'
-    static final PROJECT_KEY_ARG = 'sonar.projectKey'
-    static final MANDATORY_ARGS = [n: 'sonar.projectName', v: 'sonar.projectVersion']
+    static final MANDATORY_ARGS = [k: 'sonar.projectKey',
+                                   n: 'sonar.projectName',
+                                   v: 'sonar.projectVersion'
+    ]
+    static final IGNORED_PROPERTIES = [
```

Comment:
If we don't ignore it then msbuild will throw an exception? Won't it be ignored anyway?